ModuleInstall/PackageManager/PackageManager.php

```diff
@@ -438,29 +439,65 @@ public function validate_manifest($manifest)
         }
         $GLOBALS['log']->debug("Passed with InstallType");
         if (isset($manifest['acceptable_sugar_versions'])) {
-            $version_ok = false;
+            $GLOBALS['log']->debug("Getting AcceptableSugarVersions");
```

Comment:
You can use `LoggerManager::getLogger()` instead superglobals.

Reply:
done